packages/bpk-component-infinite-scroll/src/withInfiniteScroll.js

```diff
@@ -67,6 +68,7 @@ const propTypes = {
   initiallyLoadedElements: PropTypes.number,
   elementsPerScroll: PropTypes.number,
   dataSource: PropTypes.instanceOf(DataSource).isRequired,
+  loaderMinDisplay: PropTypes.oneOf(['small', 'half', 'full']),
```

Comment:
I think we need a defaultValue for this prop too, no? Should be `full` by default.

Reply:
true